src/libcompiler_builtins/Cargo.toml

```diff
@@ -13,3 +13,4 @@ core = { path = "../libcore" }
 
 [build-dependencies]
 gcc = "0.3.27"
+rustc-cfg = { git = "https://github.com/japaric/rustc-cfg", branch = "llvm-target" }
```

Comment:
It looks like this may be something that should be in gcc-rs, but maybe I'm missing something? Couldn't we just look at `$TARGET` like we do for other platforms?